rp_redcap/tasks.py

```diff
@@ -163,8 +163,12 @@ def run(self, project_id, **kwargs):
 
                         extra_info["missing_fields"] = ", ".join(missing_fields)
 
-                    reminders[contact.urn] = extra_info
-                    data[row["record_id"]]["redcap_reminder_count"] += 1
+                    if (
```

Comment:
I think that the drop in coverage is due to the fact that this `if` statement passes in every test. Would it be possible to test where this results in `False` and doesn't run the indented code?